app/assets/javascripts/application.js

```diff
@@ -14,7 +14,6 @@
 //= require lookup_keys
 //= require editable/bootstrap-editable
 //= require editable/rails
-//= require deprecation
```

Comment:
why remove the entire file? does it make sense to leave it as a place to add new deprecated functions in the future?

Reply:
Adding the file again if needed is easy, i would rather not keep an empty file that isn't used. I also don't mind if functions are deprecated in the existing files without moving them around for it.

Reply:
Could plugins be depending on this?

Reply:
I don't think so, this is part of the core assets. If they are using the functions in this file they should stop as they are being removed.

Reply:
I've mistaken and thought deprecation included the functions to mark it as deprecated. However, that's `tfm.tools.deprecate` so ignore my comment.